salt/modules/consul.py

```diff
@@ -2126,7 +2131,7 @@ def acl_delete(consul_url=None, **kwargs):
         ret['res'] = False
         return ret
 
-    function = 'acl/delete/{0}'.format(kwargs['id'])
+    function = 'acl/destroy/{0}'.format(kwargs['id'])
```

Comment:
If so would it be prudent to try to detect a failure with `destroy` and then retry with `delete` to see if it works?